### PR TITLE
fix(graphql-types-generator): add wrapperr for sub response in angular

### LINKED ISSUE #207: Bug in Angular codegen for GraphQL API subscriptions
refer 
https://github.com/aws-amplify/amplify-codegen/issues/25
https://github.com/aws-amplify/amplify-cli/issues/5284

creating new as all the tickets are closed and the issue still persist.

### PR DESCRIPTION
**Pull request by AaronZyLee**
#### Description of changes

Add `Subscription` type in angular type-gen and the corresponding `Pick` wrapper in subscription response

#### Issue #, if available

fix #207

#### Description of how you validated changes
`yarn test`

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-codegen/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] Breaking changes to existing customers are released behind a feature flag or major version update
- [x] Changes are tested using sample applications for all relevant platforms (iOS/android/flutter/Javascript) that use the feature added/modified

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/graphql-types-generator/src/angular/index.ts [hunk 1]
```diff
@@ -54,6 +52,38 @@ function generateTypes(generator: CodeGenerator, context: LegacyCompilerContext)
...
 
+function generateSubscriptionResponseWrapper(generator: CodeGenerator) {
+  generator.printOnNewline(`
+  export interface SubscriptionResponse<T> {
+    value: GraphQLResult<T>;
+  }`);
+  generator.printNewline();
+}
+function generateSubscriptionOperationTypes(generator: CodeGenerator, context: LegacyCompilerContext) {
+  const typeName = 'Subscription';
```
Comment (phani-srikar) [inline packages/graphql-types-generator/src/angular/index.ts:63]: Introducing `Subscription` type might have high probability to clash with developer's existing type with same name. Can we use` __Subscription` or `__SubscriptionContainer` instead?

**Review by phani-srikar — APPROVED** [review]
Suggested naming change to avoid conflicts. This should also probably go behind a FF? We can keep the FF deprecation shorter than others, probably a couple months.

File: packages/graphql-types-generator/src/angular/index.ts (hunk 1 above)
Comment (AaronZyLee) [reply packages/graphql-types-generator/src/angular/index.ts]: I think ff is not necessary because it is a fix and the original code does not work.

Comment (AaronZyLee) [reply packages/graphql-types-generator/src/angular/index.ts]: I change the type name to `__SubscriptionContainer` to avoid conflicts

**Pull request merged**